<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkMultiLabelSegmentationControls</class>
 <widget class="QWidget" name="QmitkMultiLabelSegmentationControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>844</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>MS Shell Dlg 2</family>
    <pointsize>8</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
    <underline>false</underline>
    <strikeout>false</strikeout>
   </font>
  </property>
  <property name="windowTitle">
   <string>QmitkSegmentation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QGroupBox" name="groupBox_DataSelection">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Data Selection</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_PatientImage">
        <property name="text">
         <string>Patient Image</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QmitkDataStorageComboBox" name="m_cbReferenceNodeSelector">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="2">
       <widget class="QToolButton" name="m_pbNewSegmentationSession">
        <property name="toolTip">
         <string>Create a new segmentation session</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../resources/multilabelsegmentation.qrc">
          <normaloff>:/multilabelsegmentation/NewSegmentationSession_48x48.png</normaloff>:/multilabelsegmentation/NewSegmentationSession_48x48.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="shortcut">
         <string>N</string>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_Segmentation">
        <property name="text">
         <string>Segmentation</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QmitkDataStorageComboBox" name="m_cbWorkingNodeSelector">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_Layer">
     <property name="title">
      <string>Layers</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QToolButton" name="m_btAddLayer">
        <property name="toolTip">
         <string>Add a layer to the current segmentation session</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/Qmitk/AddLayer_48x48.png</normaloff>:/Qmitk/AddLayer_48x48.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_btDeleteLayer">
        <property name="toolTip">
         <string>Delete the active layer</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/Qmitk/DeleteLayer_48x48.png</normaloff>:/Qmitk/DeleteLayer_48x48.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="m_btPreviousLayer">
        <property name="toolTip">
         <string>Change to the previous available layer</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/Qmitk/PreviousLayer_48x48.png</normaloff>:/Qmitk/PreviousLayer_48x48.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_btNextLayer">
        <property name="toolTip">
         <string>Change to the next available layer</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/Qmitk/NextLayer_48x48.png</normaloff>:/Qmitk/NextLayer_48x48.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="m_cbActiveLayer">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Switch to a layer</string>
        </property>
        <item>
         <property name="text">
          <string>0</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_Labels">
     <property name="title">
      <string>Labels</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QToolButton" name="m_pbNewLabel">
        <property name="toolTip">
         <string>Add a new label to the current segmentation session</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../resources/multilabelsegmentation.qrc">
          <normaloff>:/multilabelsegmentation/NewLabel_48x48.png</normaloff>:/multilabelsegmentation/NewLabel_48x48.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="shortcut">
         <string>N</string>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_btLockExterior">
        <property name="toolTip">
         <string>Lock/Unlock exterior</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/Qmitk/UnlockExterior_48x48.png</normaloff>
          <normalon>:/Qmitk/LockExterior_48x48.png</normalon>:/Qmitk/UnlockExterior_48x48.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="m_pbShowLabelTable">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>34</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Show a table with all labels in the current segmentation session</string>
        </property>
        <property name="text">
         <string>&gt;&gt;</string>
        </property>
        <property name="iconSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoRaise">
         <bool>false</bool>
        </property>
        <property name="arrowType">
         <enum>Qt::NoArrow</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QmitkLabelSetWidget" name="m_LabelSetWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="m_tw2DTools">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QTabWidget::tab-bar { alignment: middle; }</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="usesScrollButtons">
      <bool>true</bool>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="m_tb2DTools">
      <attribute name="title">
       <string>2D Tools</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QmitkToolGUIArea" name="m_ManualToolGUIContainer2D" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QmitkToolSelectionBox" name="m_ManualToolSelectionBox2D" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>3D Tools</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QmitkToolGUIArea" name="m_ManualToolGUIContainer3D" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QmitkToolSelectionBox" name="m_ManualToolSelectionBox3D" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_gbInterpolation">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Interpolation</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QComboBox" name="m_cbInterpolation">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Disabled</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2D Interpolation</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3D Interpolation</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="m_swInterpolation">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QmitkSliceBasedInterpolatorWidget" name="m_SliceBasedInterpolatorWidget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QmitkSurfaceBasedInterpolatorWidget" name="m_SurfaceBasedInterpolatorWidget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <zorder>m_LabelSetWidget</zorder>
  <zorder>groupBox_DataSelection</zorder>
  <zorder>m_tw2DTools</zorder>
  <zorder>m_gbInterpolation</zorder>
  <zorder>groupBox_Layer</zorder>
  <zorder>groupBox_Labels</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QmitkDataStorageComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">QmitkDataStorageComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>QmitkToolSelectionBox</class>
   <extends>QWidget</extends>
   <header location="global">QmitkToolSelectionBox.h</header>
  </customwidget>
  <customwidget>
   <class>QmitkToolGUIArea</class>
   <extends>QWidget</extends>
   <header location="global">QmitkToolGUIArea.h</header>
  </customwidget>
  <customwidget>
   <class>QmitkLabelSetWidget</class>
   <extends>QWidget</extends>
   <header location="global">Qmitk/QmitkLabelSetWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QmitkSliceBasedInterpolatorWidget</class>
   <extends>QWidget</extends>
   <header location="global">QmitkSliceBasedInterpolatorWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QmitkSurfaceBasedInterpolatorWidget</class>
   <extends>QWidget</extends>
   <header location="global">QmitkSurfaceBasedInterpolatorWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="global">QmitkToolReferenceDataSelectionBox.h</include>
  <include location="global">QmitkToolGUIArea.h</include>
  <include location="global">QmitkToolSelectionBox.h</include>
 </includes>
 <resources>
  <include location="../../resources/multilabelsegmentation.qrc"/>
 </resources>
 <connections/>
</ui>
